ansible_collections/arista/avd/roles/eos_cli_config_gen/templates/eos/ip-ssh-client-source-interfaces.j2

```diff
@@ -0,0 +1,11 @@
+{# eos - IP ssh Client Source Interfaces#}
+{% for ip_ssh_client_source_interface in ip_ssh_client_source_interfaces | arista.avd.natural_sort %}
+{%     if ip_ssh_client_source_interface.name is arista.avd.defined %}
+{%         set ip_ssh_client_cli = "ip ssh client source-interface " ~ ip_ssh_client_source_interface.name | capitalize %}
+{%         if ip_ssh_client_source_interface.vrf is arista.avd.defined %}
+{%             set ip_ssh_client_cli = ip_ssh_client_cli ~ " vrf " ~ ip_ssh_client_source_interface.vrf %}
+{%         endif %}
+!
```

Comment:
EOS output is:
```
!
ip ssh client source-interface Loopback0 vrf default
ip ssh client source-interface Management1 vrf MGMT
```
I suggest you remove this `!`. If you really want to match the first `!`, you should add an `if ip_ssh_client_source_interfaces is arista.avd.defined` above everything to output this `!` once.

Reply:
Actually, I don't want to add !, but I saw similar implementation for ip-http-client-source-interface. So I added that.

Reply:
Made changes accordingly.